types/react-autosuggest/index.d.ts

```diff
@@ -199,4 +208,13 @@ declare namespace Autosuggest {
          */
         theme?: Theme;
     }
+
+    interface AutosuggestState<TSuggestion> {
+        isFocused: boolean;
+        isCollapsed: boolean;
+        highlightedSectionIndex: number | null;
+        highlightedSuggestionIndex: number | null;
+        highlightedSuggestion: TSuggestion | null;
+        valueBeforeUpDown: TSuggestion | null;
```

Comment:
I'd make these `undefined` as well, unless you're sure they'll actually be `null`'s

Reply:
These are indeed initialized as `null` in the codebase. See https://github.com/moroshko/react-autosuggest/blob/d626cef720df0ed5a4c7df4d539836b86bda6afa/src/Autosuggest.js#L101-L108